shellclear/src/bin/cmd/clear.rs

```diff
@@ -31,17 +39,18 @@ pub fn run(
                             "could not backup shell {:?} history. err: {:?}",
                             shell_context.history.shell, e
                         )),
-                    })
+                    });
                 }
             }
         }
     }
 
-    let findings = en.find_history_commands_from_shall_list(shells_context, true)?;
-
-    let sensitive_commands = findings.get_sensitive_commands();
+    let (findings, sensitive_commands) =
+        en.find_history_commands_from_shell_list(shells_context)?;
     let emojis = Emojis::default();
 
+    Clearer::default().write_findings(shells_context, &findings, matches.is_present("remove"))?;
```

Comment:
his is and purpose for use `Clearer::default()`? it look like this struct is empty.
maybe just call it `Clearer::write_findings`